media-sound/pulseaudio-modules-bt/pulseaudio-modules-bt-1.3-r1.ebuild

```diff
@@ -16,16 +16,16 @@ SRC_URI="
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
+IUSE="ldac ofono-headset fdk ffmpeg native-headset"
```

Comment:
Let me alphabetize them, sounds like a good practice.

About the change, is that a good practice? Likely I can enable encoders by default, but backends doesn't make much sense as they make likely same (different technology, same feature)

Pulseaudio provides a elog if you enable both at same time:

if use native-headset && use ofono-headset; then
        elog "You have enabled both native and ofono headset profiles. The runtime decision"
        elog "which to use is done via the 'headset' argument of module-bluetooth-discover."
    fi

Reply:
fdk not default for a reason?

Reply:
@zx2c4 Yeah, the reason why I didn't included it by default, is because AAC encoder( FDK use flag) requires third party Fraunhofer AAC library, which is non-free opensource project extracted from AOSP (Android). Users may require explicit license acceptance in order for that. Users using FDK likely are already using explicit USE flag for FDK, as non-default for ffmpeg or obs-studio.

Do you think is okay to include it enabled by default? As you might have more experience applying changes, I would take your feedback here.